InvenTree/common/models.py

```diff
@@ -1664,6 +1664,10 @@ class Meta:
         help_text=_('Unit price at specified quantity'),
     )
 
+    updated = models.DateTimeField(
+        auto_now=True, verbose_name=_("last updated")
```

Comment:
Suggestion(nitpick - non-blocking): If you put each property on a separate line and add a komma at the end future diffs will be smaller / cleaner to read.

Reply:
Yeah, this is what my default black settings produced.  I will need to see if there is an option that will fix this (or I need to change my formatter to whatever Inventree uses).